### PR TITLE
I18n for views/student_identifier_types

### PR DESCRIPTION
**Pull request by jiteshjha**
Hi @tarebyte,
One more PR for student_identifier_types strings.

Thank you!

### CODE REVIEW

**Comment by CGA1123** [comment]
#662

File: app/views/student_identifier_types/_student_identifier_type.html.erb [hunk 1]
```diff
@@ -15,13 +15,13 @@
 
 <div class="remodal text-left" data-remodal-id="<%= "delete-identifier-type-#{student_identifier_type.id}" %>">
   <button data-remodal-action="close" class="remodal-close"><%= octicon('x') %></button>
-  <h2 class="remodal-header">Are you sure you want to delete this identifier type?</h2>
+  <h2 class="remodal-header"><%= I18n.t('views.student_identifier_types.are_you_sure_delete_identifier_type') %></h2>
```
Comment (tarebyte) [inline app/views/student_identifier_types/_student_identifier_type.html.erb]: Instead of using `I18n.t` can we use just use `t`?

Comment (jiteshjha) [inline app/views/student_identifier_types/_student_identifier_type.html.erb]: Yes, we can! I've made the changes. :)

**Comment by nwoodthorpe** [comment]
This is still valid, we just haven't had time to review it.

**Pull request closed without merging**